build.gradle

```diff
@@ -329,6 +374,32 @@ subprojects {
     duplicatesStrategy = DuplicatesStrategy.FAIL
     exclude('**/*.xml')
   }
+
+  def publishRegularArtifacts = projectsToPublish.containsKey(project.path)
+
+  if (publishRegularArtifacts) {
+    task sourceJar(type: Jar) {
+      from sourceSets.main.allJava
+      classifier "sources"
+    }
+  }
+
+  // TODO: Enable after we have valid javadocs
```

Comment:
We should fix that some time soon... what is the issue? Invalid JavaDoc syntax? Or missing annotations?

Reply:
Yeah, it looks like invalid syntax